### PR TITLE
Enable JIT on AArch64 macOS

### PR DESCRIPTION
**Pull request by knn-k**
This commit enables JIT on AArch64 macOS.

Signed-off-by: KONNO Kazuhiro <konno@jp.ibm.com>

### CODE REVIEW

**Comment by knn-k** [comment]
- Related PR for JDK 18 build: ibmruntimes/openj9-openjdk-jdk18#2

**Review by pshipton — APPROVED** [review]

File: make/autoconf/flags.m4 [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #
 # Copyright (c) 2011, 2018, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2021, Azul Systems, Inc. All rights reserved.
```
Comment (keithc-ca) [inline make/autoconf/flags.m4:3]: Why was this line added? It's not upstream.

Comment (knn-k) [reply make/autoconf/flags.m4:3]: I took the `MACOSX_VERSION_MIN` code from https://github.com/ibmruntimes/openj9-openjdk-jdk17/commit/e5ed92f29af6833890c1d511b5120e7e71bbf0c5 .
I thought the Azul copyright line was in the same file, but it seems I misunderstood it.
The latest flags.m4 in openj9-openjdk-jdk17 does not have the additional copyright line.
I am going to open a new PR to remove the line.

Comment (knn-k) [reply make/autoconf/flags.m4:3]: I opened #475.

Comment (keithc-ca) [reply make/autoconf/flags.m4:3]: Merged. Thanks.
